File: Python/Exercicio18.py
"""
Faça um algoritmo que conte de 1 a 100 e a cada múltiplo de 10 emita
uma mensagem: "Múltiplo de 10"
"""

n = 100
x = 0

while x <= 100:
    x += 1
    if x % 10 == 0:
        print("{} é multiplo de 10".format(x))
 

File: C/Exercicio18.c
/*
Faca um algoritmo que conte de 1 a 100 e a cada multiplo de 10 emita
uma mensagem: "Multiplo de 10"
*/

#include<stdio.h>

int main(){
	
	int n = 0;
	
	while(n <= 100){
		n += 1;
		if(n % 10 == 0){
			printf("%d e multiplo de 10\n", n);
		}
	}
}
